docs/sources/administration/configuration.md

```diff
@@ -202,15 +202,14 @@ Or redirect port 80 to the Grafana port using:
 $ sudo iptables -t nat -A PREROUTING -p tcp --dport 80 -j REDIRECT --to-port 3000
 ```
 
-Another way is put a webserver like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
+Another way is to put a web server like Nginx or Apache in front of Grafana and have them proxy requests to Grafana.
```

Comment:
Currently, there is no entry for that in the style guide, and I too vote to add it as web server. Rationale being that web page is not written webpage, and web browser is not webbrowser. Will handle that in a subsequent PR.

Reply:
Moving my comment to in here. Do we have definitive guidance on using "webserver" versus "web server" in our style guide? If not, we should probably add instructions to use "web server" and not "webserver".

I will update the style guide.